model_compression_toolkit/target_platform_capabilities/tpc_models/imx500_tpc/v2/tp_model.py

```diff
@@ -151,6 +152,14 @@ def generate_tp_model(default_config: OpQuantizationConfig,
     # this configuration will be used for the operation quantization:
     default_configuration_options = tp.QuantizationConfigOptions([default_config])
 
+    # const configuration is similar to the default eight bit configuration except for PoT
+    # quantization method for the constant.
+    const_config = default_config.clone_and_edit(
+        default_weight_attr_config=default_config.default_weight_attr_config.clone_and_edit(
+            weights_quantization_method=tp.QuantizationMethod.POWER_OF_TWO,
+            enable_weights_quantization=True))
```

Comment:
Is the quantization per-channel or per-tensor?

Reply:
Following this comment:
- It should be per-tensor.
- you need to make sure that you are copying `eight_bits_default` config not `linear_eight_bits`.
- I think that the `weights_quantization_method` in `default_weight_attr_config`  is already defined to be POWER_OF_TWO, so you don't need to edit it (verify this before changing). Also the per-tensor property.